addons/mrp/tests/test_stock_report.py

```diff
@@ -327,3 +327,37 @@ def test_subkit_in_delivery_slip(self):
 
 
         self.assertFalse(keys, "All keys should be in the report with the defined order")
+
+    def test_mo_overview(self):
+        """ Test that the overview does not traceback when the final produced qty is 0
+        """
+        product_chocolate = self.env['product.product'].create({
+            'name': 'Chocolate',
+            'type': 'consu',
+        })
+        product_chococake = self.env['product.product'].create({
+            'name': 'Choco Cake',
+            'type': 'product',
```

Comment:
From 17.3, this type does not exist anymore (cf 728d9f83f6d140b57b91000f683298f1f5bd43c9)